src/biz/metrics.go

```diff
@@ -234,12 +242,30 @@ func (mc *MetricsFetcher) memory(mem raw.Memory) Memory {
 		usagePercent = 100 * float64(usage) / float64(memLimits)
 	}
 
+	swapLimit := mem.SwapLimit
+	if mem.SwapLimit > math.MaxInt64/2 {
+		swapLimit = 0
+	}
+	swapUsagePercent := float64(0)
+	if swapLimit > 0 {
+		swapUsagePercent = 100 * float64(mem.SwapUsage+mem.RSS) / float64(swapLimit)
```

Comment:
Why are we adding `mem.RSS` to `mem.SwapUsage`? I'd expect those to be unrelated.

Reply:
Good point It's a bit confusing I will add a comment and perhaps I should change the metric name of swapLimit to add more clarity